apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudTrace/TraceConfiguration.cs

```diff
@@ -21,15 +21,15 @@ namespace Google.Devtools.AspNet
     /// </summary>
     public sealed class TraceConfiguration
     {
-        public static readonly double DefaultQpsSampleRate = 1.0;
+        public const double DefaultQpsSampleRate = 1.0;
 
-        private readonly double _qpsSampleRate;
+        public readonly double QpsSampleRate;
```

Comment:
Avoid public fields. Use `public double QpsSampleRate { get; }` instead. Also, this needs /// docs. (same comments for all others like this).